users/ignoring.rst

```diff
@@ -34,62 +34,77 @@ Patterns
+|             |           |          | from the named file. It is an error for a file to not exist or be             |
+|             |           |          | included more than once. Note that while this can be used to include          |
+|             |           |          | patterns from a file in a subdirectory, the patterns themselves are           |
+|             |           |          | still relative to the folder *root*. Example:                                 |
+|             |           |          | ``#include more-patterns.txt``.                                               |
+|             |           |          |                                                                               |
++-------------+-----------+----------+-------------------------------------------------------------------------------+
+| ``*``       | Anywhere  | All      | Asterisk matches zero or more characters in a filename, but does not          |
+|             |           |          | match the directory separator. ``te*st`` matches ``test``,                    |
+|             |           |          | ``subdir/telerest`` but not ``tele/rest``.                                    |
+|             |           |          |                                                                               |
++-------------+-----------+----------+-------------------------------------------------------------------------------+
+| ``**``      | Anywhere  | All      | Double asterisk matches as above, but also directory separators.              |
+|             |           |          | ``te**st`` matches ``test``, ``subdir/telerest`` and                          |
+|             |           |          | ``tele/sub/dir/rest``.                                                        |
+|             |           |          |                                                                               |
++-------------+-----------+----------+-------------------------------------------------------------------------------+
+| ``?``       | Anywhere  | All      | Question mark matches a single character that is not the directory            |
+|             |           |          | separator. ``te??st`` matches ``tebest`` but not ``teb/st`` or                |
+|             |           |          | ``test``.                                                                     |
+|             |           |          |                                                                               |
++-------------+-----------+----------+-------------------------------------------------------------------------------+
+| ``[]``      | Anywhere  | All      | Characters enclosed in square brackets ``[]`` are interpreted as a character  |
+|             |           |          | range ``[a-z]``. Before using this syntax you should have a basic             |
+|             |           |          | understanding of regular expression character classes.                        |
```

Comment:
I know this is not your sentence, but I'd still remove it while at it: It insinuates globbing supports character classes (`\s`, `\W` or similar) while I am pretty sure it does not. And just generally I'd like to not mention regex here, as our syntax is explicitly not regex.